sudo-enforce-allowed-users/sudo-enforce-allowed-users.cf

```diff
@@ -23,7 +23,7 @@ bundle agent sudo_enforce_allowed_users
         slist => { "ALL ALL=(ALL) ALL",
                    "ALL ALL=(ALL:ALL) ALL" };
 
-      "_check_command" string => "visudo --quiet --check --strict --file /etc/sudoers";
+      "_check_command" string => "/usr/sbin/visudo --quiet --check --strict --file /etc/sudoers";
```

Comment:
Shouldn't we just use useshell for the commands promise? More likely to succeed. Maybe even add a class about whether visudo is available?

Reply:
Ah right, I see in https://github.com/nickanderson/cfengine-security-hardening/issues/14 that visudo is present but not in PATH so sure, let's roll with this path for now. Later we can add things to stdlib paths if we need. :+1: